scripts/onboarding/aks/multi-tenancy-multiclusters/existingClusterParam.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "resourceGroupName": {
+      "value": "<resourceGroupName>"
+    },
+    "location": {
```

Comment:
since config DCE must be in  same region as AKS cluster and this needs to be removed from shared template.